scripts/release.sh

```diff
@@ -59,5 +64,11 @@ for runtime in "${runtimes[@]}"; do
     artifacts+=("${artifact_name}")
 done
 
-# Create a pre-release using GitHub CLI
-gh release create --title "${tag_name}" --draft "${tag_name}" "${artifacts[@]}" "${checksum_name}"
+# Create a release using GitHub CLI
+if [[ "$tag_name" == "-" ]]; then
```

Comment:
`$ [[ "v1.0.0-rc.1" == "-" ]] && echo yes` prints nothing. It seems the condition expression is not correct.